easybuild/easyconfigs/l/LAMMPS/LAMMPS-3Mar2020-foss-2020a-Python-3.8.2-kokkos.eb

```diff
@@ -0,0 +1,92 @@
+name = 'LAMMPS'
+version = '3Mar2020'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-kokkos'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++11'}
+
+# 'https://github.com/lammps/lammps/archive/'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['stable_%(version)s.tar.gz']
+patches = ['lammps-stable_3Mar2020_hack_openmp_gcc9.patch']
+checksums = [
+    'a1a2e3e763ef5baecea258732518d75775639db26e60af1634ab385ed89224d1',  # stable_3Mar2020.tar.gz
+    '41a0bcb828be22d38bb489bbd4b1fd7803d7771a2308371f01e961c52b8c869f',  # lammps-stable_3Mar2020_hack_openmp_gcc9.patch
+]
+
+# See https://github.com/lammps/lammps/pull/1483 for why this is needed
```

Comment:
this comment is misplaced, should be above `patches = `?

Reply:
Ah, inject checksums did this